.editorconfig

```diff
@@ -233,6 +233,10 @@ dotnet_diagnostic.IDE0072.severity = silent
 # IDE0083: Use pattern matching
 dotnet_diagnostic.IDE0083.severity = silent
 
+# IDE0090: Use 'new(...)'
+# Remove this once https://github.com/dotnet/roslyn-analyzers/issues/5867 is fixed and fix violations.
```

Comment:
Can we file a separate issue? Or can we wait for the new Microsoft.CodeAnalysis.Analyzers pre-release package with #6119 merged and update the repo to consume that before merging this PR?

Reply:
I'm okay with waiting until #6119 is merged.

Reply:
@Youssef1313 It is merged now. You can wait for new version showing up [here](https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet7/NuGet/Microsoft.CodeAnalysis.Analyzers/versions)

Reply:
You'll have to update https://github.com/dotnet/roslyn-analyzers/blob/5463e16b5c0d1a83729babf5772de8b2917195c9/eng/Versions.props#L52

Reply:
Or probably just bump https://github.com/dotnet/roslyn-analyzers/blob/5463e16b5c0d1a83729babf5772de8b2917195c9/eng/Versions.props#L49